tools/hangar/__snapshots__/test_corpus/for_loop.w_compile_tf-aws.md

```diff
@@ -1,5 +1,24 @@
 # [for_loop.w](../../../../examples/tests/valid/for_loop.w) | compile | tf-aws
```

Comment:
@MarkMcCulloh I am in love with these markdown snapshots! Just sayin...

Reply:
Right? I've been thinking about proposing this as a standard format in the vitest repo